delivery_carrier_max_weight_constraint/models/stock_quant_package.py

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class StockQuantPackage(models.Model):
+    _inherit = "stock.quant.package"
+
+    @api.constrains("shipping_weight", "packaging_id")
+    def _constrain_shipping_weight(self):
```

Comment:
```suggestion
    def _check_shipping_weight(self):
```

See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models